components/server/src/api/workspace-service-api.ts

```diff
@@ -63,4 +67,30 @@ export class WorkspaceServiceAPI implements ServiceImpl<typeof WorkspaceServiceI
             yield response;
         }
     }
+
+    async listWorkspaces(req: ListWorkspacesRequest, context: HandlerContext): Promise<ListWorkspacesResponse> {
+        // TODO: pagination check - max min pageSize ...
+        // TODO: implement req.scope req.sorts
```

Comment:
Does it mean we don't need sorting yet and can remove from api all together?

Reply:
Yes we can remove it, since we don't want to implement it yet with papi, it's just a proposal of https://github.com/gitpod-io/gitpod/pull/19022#discussion_r1385507978